nmdc_server/migrations/versions/afa1ff687968_convert_packagename_to_an_array.py

```diff
@@ -0,0 +1,57 @@
+"""Convert packageName to an array
+
+Revision ID: afa1ff687968
+Revises: 317274ad8137
+Create Date: 2024-08-28 22:27:28.622234
+
+"""
+
+from typing import Optional
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.sql import column, table
+
+# revision identifiers, used by Alembic.
+revision: str = "afa1ff687968"
+down_revision: Optional[str] = "317274ad8137"
+branch_labels: Optional[str] = None
+depends_on: Optional[str] = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    submission_metadata = table(
+        "submission_metadata",
+        column("id", sa.String),
+        column("metadata_submission", JSONB),
+        column("study_name", sa.String),
+        column("templates", JSONB),
+    )
+
+    connection = op.get_bind()
+    submissions = connection.execute(
+        sa.select([submission_metadata.c.id, submission_metadata.c.metadata_submission])
+    )
+
+    for submission in submissions:
+        metadata_submission = submission.metadata_submission
+        package_name = metadata_submission.get("packageName")
```

Comment:
Looks like we do add a default value to the package name from the frontend during submission creation. Also, there are no `null`s in any submission on dev, so this seems safe.